### DISCUSSION TITLE
React support?

### DISCUSSION THREAD

**Discussion by baptisteArno** [histoire-dev/histoire#199] [General]
Are you planning on supporting React?

**Comment by brielov** [comment]
Same here, are there any plans?

**Comment by jimburbfs** [comment]
Getting a bump on this.

**Comment by airtonix** [comment]
annoying +1 to further unfuriate the developer into never responding.

**Comment by marek-hanzal** [comment]
I don't _believe_ I've found a project **without** React support. Makes sense, when there is no **React**ion.

That's sad, because project looks good :(

**Comment by alexn-s** [comment]
i would love react support. histoire looks really neat

**Reply by marek-hanzal** [reply]
Bad luck, [Nextra](https://nextra.site/) is the way to go and it works great; more for ones already using Next.js. From the time I've commented here I've tried some other solutions and Nextra is the only one which works as expected. Just to keep things clear: I've nothing to do with them, just using the tool.
